### ISSUE TITLE
Recent change to use the Google App Engine Admin API breaks file mimetypes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#307]
```
What steps will reproduce the problem?

Perform a gcloud preview app deploy app.yaml


What is the expected output? What do you see instead?

The website that was deployed using an earlier version of gcloud (that does not 
use the Google App Engine Admin API). Instead, files are served with the header 
Content-Type: application/octet-stream which will make the website break (CSS 
doesn't load with that mimetype).


What is the output of 'gcloud info'?

Google Cloud SDK [0.9.82]

Platform: [Mac OS X, x86_64]
Python Version: [2.7.10 (default, Jul 13 2015, 12:05:58)  [GCC 4.2.1 Compatible 
Apple LLVM 6.1.0 (clang-602.0.53)]]
Site Packages: [Disabled]

Installation Root: 
[/opt/homebrew-cask/Caskroom/google-cloud-sdk/latest/google-cloud-sdk]
Installed Components:
  core: [2015.10.08]
  app-engine-python: [1.9.27]
  core-nix: [2015.09.03]
  app-engine-php: [ ]
  app-engine-java: [1.9.27]
  gcloud: [2015.10.08]
  gsutil-nix: [4.14]
  app-engine-python-extras: [1.9.21]
  gsutil: [4.15]
  bq: [2.0.18]
  app-engine-php-darwin: [2015.08.17]
  bq-nix: [2.0.18]
  kubectl: []
  app: [2015.10.08]
  kubectl-darwin-x86_64: [1.0.6]
System PATH: 
[/Users/blixt/src/torch/install/bin:/opt/homebrew-cask/Caskroom/google-cloud-sdk
/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X
11/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Users/blixt/Do
wnloads/dart/dart-sdk/bin:/Users/blixt/.rvm/bin:/usr/local/go/bin:/Users/blixt/g
o/bin:/Users/blixt/android/sdk/tools:/Users/blixt/android/sdk/platform-tools:/Us
ers/blixt/android/sdk/build-tools/19.1.0]
Cloud SDK on PATH: [True]

Installation Properties: 
[/opt/homebrew-cask/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/properties
]
User Config Directory: [/Users/blixt/.config/gcloud]
User Properties: [/Users/blixt/.config/gcloud/properties]
Current Workspace: [/Users/blixt/src/roger-web-client]
Workspace Config Directory: [/Users/blixt/src/roger-web-client/.gcloud]
Workspace Properties: [/Users/blixt/src/roger-web-client/.gcloud/properties]

Account: [blixt@47center.com]
Project: [roger-web-client]

Current Properties:
  [core]
    project: [roger-web-client]
    account: [blixt@47center.com]
    disable_usage_reporting: [True]
  [app]
    use_appengine_api: [True]
    suppress_change_warning: [true]

Logs Directory: [/Users/blixt/.config/gcloud/logs]
Last Log File: [/Users/blixt/.config/gcloud/logs/2015.10.14/20.59.16.869333.log]
```

Original issue reported on code.google.com by `m...@blixt.org` on 15 Oct 2015 at 1:05

**Comment by GoogleCodeExporter** [comment]
```
To clarify, the .yaml file needs a static_dir or static_files directive without 
the mimetype specified for this to happen.
```

Original comment by `m...@blixt.org` on 15 Oct 2015 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
For people stuck with this issue, the solution is to revert to not use the 
Google App Engine Admin API:

gcloud config set app/use_appengine_api false
```

Original comment by `m...@blixt.org` on 15 Oct 2015 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
I had this issue too! SUX
```

Original comment by `ric...@gmail.com` on 15 Oct 2015 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. We're looking into this now.

```

Original comment by `dlor...@google.com` on 15 Oct 2015 at 2:04

**Comment by GoogleCodeExporter** [comment]
```
Fix should go out in 0.9.83 release around 10/21.
```

Original comment by `che...@google.com` on 16 Oct 2015 at 1:46
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Is this fixed yet as of the latest 0.9.83 release today?
```

Original comment by `markus.o...@gmail.com` on 21 Oct 2015 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
Yes, this should be fixed as of the 0.9.83 release. Please let us know if 
you're still seeing any issues.
```

Original comment by `dlor...@google.com` on 21 Oct 2015 at 5:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I just tested this and it now works (for a CSS file I now get "Content-Type: 
text/css" instead of "… application/octet-stream").
```

Original comment by `m...@blixt.org` on 21 Oct 2015 at 5:55
